libraries/node-core-library/src/IPackageJson.ts

```diff
@@ -71,6 +71,77 @@ export interface IDependenciesMetaTable {
   };
 }
 
+/**
+ * This interface is part of the {@link IPackageJson} file format. It is used for the values
+ * of the "exports" field.
+ *
+ * See {@link https://nodejs.org/api/packages.html#conditional-exports | Node.js documentation on Conditional Exports} and
+ * {@link https://nodejs.org/api/packages.html#community-conditions-definitions | Node.js documentation on Community Conditional Exports}.
+ *
+ * Documentation for fields in this interface is adapted from the Node.js documentation.
+ *
+ * @public
+ */
+export interface IPackageJsonExports {
```

Comment:
Note that conditions are allowed to be arbitrary user-defined strings, as well. They can also be nested.

Reply:
In these typings, I think we should only list well-known values. We aren't validating the schema.